lib/min_heap.rb

```diff
@@ -8,24 +8,31 @@ def initialize(key, value)
 end
 
 class MinHeap
-
   def initialize
     @store = []
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity:ologn
+  # Space Complexity: o1
   def add(key, value = key)
```

Comment:
👍 , Since `heap_up` is a recursive method, you have a space complexity incurred.